src/gpl/src/nesterovBase.cpp

```diff
@@ -1922,14 +1922,17 @@ void NesterovBase::updateAreas()
                           / static_cast<float>(whiteSpaceArea_);
 
   if (totalFillerArea_ < 0) {
-    log_->error(GPL,
-                303,
-                "Use a higher -density or "
-                "re-floorplan with a larger core area.\n"
-                "Given target density: {:.2f}\n"
-                "Suggested target density: {:.2f}",
-                targetDensity_,
-                uniformTargetDensity_);
+    log_->warn(
+        GPL,
+        303,
+        "Not enough movable area, negative totalFillerArea_ in updateAreas(). "
+        "Given target density: {:.2f}. "
+        "targetDensity set with uniform density: {:.2f}",
```

Comment:
Same here. I prefer a more direct message, showing that the target density will change and explaining why it is necessary.